validator/api/src/main/java/tech/pegasys/teku/validator/api/GraffitiParser.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.validator.api;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.apache.tuweni.bytes.Bytes32;
+
+public class GraffitiParser {
+  public static Bytes32 loadFromFile(final Path graffitiFile) throws GraffitiLoaderException {
+    try {
+      checkNotNull(graffitiFile, "GraffitiFile path cannot be null");
+      if (Files.size(graffitiFile) > 32) {
+        throw new GraffitiLoaderException("GraffitiFile size too big, maximum size is 32 bytes");
+      }
```

Comment:
nit: could read the file size to variable and add context to the message, so it could be 'Graffiti file is too big (33 bytes), maximum size is 32 bytes'

Reply:
This relates to when/if we strip whitespace so won't directly address here yet